master-firmware/src/base/map_server.cpp

```diff
@@ -0,0 +1,70 @@
+#include <ch.h>
+#include <hal.h>
+#include <string.h>
+#include <error/error.h>
+#include <timestamp/timestamp.h>
+
+#include "config.h"
+#include "main.h"
+#include "priorities.h"
+
+#include "base/base_controller.h"
+#include "base/map.h"
+#include "base/map_server.h"
+#include "robot_helpers/beacon_helpers.h"
+#include "robot_helpers/trajectory_helpers.h"
+#include "strategy/state.h"
+
+#define MAP_SERVER_STACKSIZE 1024
+
+static THD_FUNCTION(map_server_thd, arg)
+{
+    enum strat_color_t color = *(enum strat_color_t*)arg;
+    chRegSetThreadName(__FUNCTION__);
+
+    const int robot_size = config_get_integer("master/robot_size_x_mm");
```

Comment:
Maybe move this in the while() so that it can be changed using the parameter API.

Reply:
done